### PR TITLE
feat: semantic html doc

### PR DESCRIPTION
**Pull request by andron13**
## What type of PR is this? (select all that apply)

- [x ] 📝 Documentation Update

#42

### CODE REVIEW

File: readme/semantic-webaccessibility.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    </main>
+  </article>
+</section>
+```
+
+**A** tag could be replaced by **Link**. But this is just an example of how one could do it. We are only talking about layout here.
+
+Please at least use semantic markup and remember, we have countless opportunities to express our thoughts.
+
+Use elements such as header, nav, main, section, article, p, aside, footer and ect to accurately reflect the meaning and purpose of the content they contain.
```
Comment (SpaNb4) [inline readme/semantic-webaccessibility.md:87]: typo: and etc

Comment (SpaNb4) [inline readme/semantic-webaccessibility.md:87]: What about highlighting the tags? `header`, `nav`, `main`

Comment (andron13) [reply readme/semantic-webaccessibility.md]: > typo: and etc

done

Comment (andron13) [reply readme/semantic-webaccessibility.md]: > What about highlighting the tags? `header`, `nav`, `main`

done

File: readme/semantic-webaccessibility.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+This markup could do with some refining. For instance, this block of code appears to contain at least two headings. Why not highlight them with `<div className="title">Upcoming courses</div>`? If there's a header and there's content, could we make use of `article`?
+
+Let's make some corrections:
+
+- The `<div>` containing the courses was changed to `<section>`. This is a relevant section for users and search engines.
+- The `<div>` containing the course details was swapped out for an `<article>`. Each course is an individual piece of content.
+- The `<div>` that served as the title has been changed to a `<header>`.
+- The `<div>` with the list of courses is now a `<main>` element as it's the unique content on the page.
```
[the remark this answers is not in the document]
Comment (andron13) [reply readme/semantic-webaccessibility.md]: > Probably not the best idea to overuse the `main` tag. In context of our website the upcoming courses is not a dominant content.
> 
> image
> 
> image
> 
> https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element

done. but it is just example. i would refactoring double section with the same css-class and split this component to item and list.

File: readme/semantic-webaccessibility.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+```jsx
+<section className="courses container">
+  <article className="courses content">
+    <header className="title">Upcoming courses</header>
+    <main className="column-2">
+      <section className="courses">
+        {courses.map(({ title, language, startDate, href, icon }) => (
+          <article key={title} className="course-card">
+            <div className="icon-container">{icon}</div>
```
[the remark this answers is not in the document]
Comment (andron13) [reply readme/semantic-webaccessibility.md]: done

**Review by ramzes-84 — APPROVED** [review]

**Review by Quiddlee — APPROVED** [review]

**Pull request merged**